kornia/filters/bilateral.py

```diff
@@ -0,0 +1,77 @@
+from __future__ import annotations
+
+import torch.nn.functional as F
+
+from kornia.core import Module, Tensor
+from kornia.core.check import KORNIA_CHECK_IS_TENSOR, KORNIA_CHECK_SHAPE
+
+from .gaussian import get_gaussian_kernel2d
+from .kernels import _unpack_2d_ks
+from .median import _compute_zero_padding
+
+
+def bilateral_blur(
+    input: Tensor,
+    kernel_size: tuple[int, int] | int,
+    sigma_color: float,
+    sigma_space: tuple[float, float],
+    border_type: str = 'reflect',
+    color_distance_type: str = "l1",
+) -> Tensor:
+    KORNIA_CHECK_IS_TENSOR(input)
+    KORNIA_CHECK_SHAPE(input, ['B', 'C', 'H', 'W'])
+
+    kx, ky = _unpack_2d_ks(kernel_size)
+    pad_x, pad_y = _compute_zero_padding(kernel_size)
+
+    padded = F.pad(input, (pad_x, pad_x, pad_y, pad_y), mode=border_type)
```

Comment:
we have exposed `pad` in core as a common used ops
https://github.com/kornia/kornia/blob/master/kornia/core/__init__.py#L13